docs/api/IoT.js-API-Stream.md

```diff
@@ -127,6 +156,18 @@ Writable stream is abstraction for target that you can write data to.
 This method writes `chunk` of data to the underlying system, when the data is flush it calls back the `callback` function.
 If you can write right after calling this method, it will return `true`, otherwise, return `false`.
 
+### `writable._readyToWrite()`
+**This method is only for implementing a Writeable stream.**
+Concrete stream implementations should call this method to inform the stream when it is ready to be written to.
+
+### `writable._write(chunk, callback, onwrite)`
+* `chunk <String> | <Buffer>` - The data to be written by the stream
+* `callback <Function()>` - Callback to be called when the chunk of data is flushed.
+* `onwrite <Function()>` - Internal Callback to be called for when the chunk of data is flushed.
+
+**This method is only for implementing a Writeable stream.**
+Concrete stream implementations should override this method. This method will be called when the stream neeeds to write a `chunk` of data. After the data is flushed, call `callback` and `onwrite` callbacks.
```

Comment:
This function is an abstract interface. It can never be called from `writeable` object. I think you need to emphasize this.